opensir/models/model.py

```diff
@@ -217,8 +220,12 @@ def fit(self, t_obs, n_obs, fit_index=None):
 
         # Check consistent inputs for fitting
         # for i in
-
+        # Set post-fit model attributes
         self.fit_index = fit_index
+        if self.postreg["t_obs"] is None:
+            self.postreg["t_obs"] = t_obs
```

Comment:
why are we storing `t_obs` and `n_obs`? In practice the user already have pointers to these arrays. What's the benefit of storing it in the model?